✨(marsha) add cronjob for management command check_harvested

lunika opened the pull request:
## Purpose

The management command check_harvested should be executed once a day.
For this we add a cronjob in marsha app configured to run the management
command every day at 3am

## Proposal

- [x] add cronjob for management command check_harvested

lunika committed:
✨(marsha) add cronjob for management command check_harvested

The management command check_harvested should be executed once a day.
For this we add a cronjob in marsha app configured to run the management
command every day at 3am

sampaccoud commented on apps/marsha/templates/services/app/cronjob_check_harvested.yml.j2:
```diff
@@ -0,0 +1,69 @@
+          labels:
+            app: marsha
+            service: app
+            version: "{{ marsha_image_tag }}"
```
I think we stopped putting this label?

lunika replied:
Ok I didn't know. But we are still using them in marsha at least.

sampaccoud replied:
that's part of @madmatah's recent work.

madmatah replied:
The label `version` has been removed on `Deployment`s (in `spec.selector` section ) and in `Static Services`.
Its use has not changed for the rest of the k8s entities.

sampaccoud commented on apps/marsha/templates/services/app/cronjob_check_harvested.yml.j2:
```diff
@@ -0,0 +1,69 @@
+                  value: "marsha-{{ marsha_postgresql_host }}-{{ deployment_stamp }}"
+                - name: POSTGRES_PORT
+                  value: "{{ marsha_postgresql_port }}"
+                - name: DJANGO_ALLOWED_HOSTS
+                  value: "{{ marsha_host | blue_green_hosts }}"
+                - name: DJANGO_CLOUDFRONT_PRIVATE_KEY_PATH
+                  value: "{{ marsha_cloudfront_private_key_path }}"
```
you don't need these 2 but ok if you want the template to the same on all jobs.

lunika replied:
If I remember, this setting is mandatory. Running the management command will saying that this setting is missing.

sampaccoud replied:
ok then :man_shrugging:

madmatah commented on apps/marsha/templates/services/app/cronjob_check_harvested.yml.j2:
```diff
@@ -0,0 +1,69 @@
+  name: "check-harvested-{{ deployment_stamp }}"
+  namespace: "{{ project_name }}"
+spec:
+  schedule: "0 3 * * *"
```
Shouldn't you introduce a new variable to make this scheduling configurable?

lunika replied:
Yes but I should do it in the other cronjob too

sampaccoud commented on CHANGELOG.md:
```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
+### Added
+
+- Marsha: cronjob running check_harvested management command
```
Don't forget to port this PR to @madmatah's branch https://github.com/openfun/arnold/tree/migration-to-k8s to avoid a regression.

lunika replied:
Yes I plan to do it.

lunika committed:
✨(marsha) allow to configre check_live_state schedule cronjob

We introduce a new variable marsha_check_live_state_cronjob_schedule
allowing to configure the schedule for the check_live_state cronjob
